composer.json

```diff
@@ -23,7 +23,7 @@
 
     "require": {
         "php":                      "^7.2, <7.4",
```

Comment:
Do we need to change it to smth like `"^7.2, 7.4.0",` in order to run on PHP 7.4.0?

Reply:
just `^7.2 <7.5`

Reply:
I think just `^7.2` is even better.

Reply:
@gep no, PHP doesn't use server (otherwise this PR wouldn't have been necessary)